resources/compass/charts/director/values.yaml

```diff
@@ -11,7 +11,9 @@ deployment:
     maxOpenConnections: 30
     maxIdleConnections: 2
 
-staticGroupNames:
+adminGroupNames:
   - "mps-superadmin"
   - "runtimeAdmin"
   - "runtimeNamespaceAdmin"
+
+operatorGroupName: "runtimeOperator"
```

Comment:
Hmm, maybe we could also transform it to an array? It would be consistent with admin group names defined above.

Like:

```yaml
operatorGroupNames: 
   - ""runtimeOperator"
```

What do you think?